### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by pymongo**

### CODE REVIEW

**Comment by limitedeternity** [comment]
@pymongo 
Screenshot_20211209-144301_Chrome

**Comment by pymongo** [comment]
@limitedeternity

Thanks for your review, I should rename Intergrade to Integrate

File: docs/en/how-integrate-with-tower.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: How to Intergrate with Tower
+title: How to Integrate with Tower
 date: 2021-11-24 20:00:00
 author: ihciah
 ---
 
-# How to Intergrate with Tower
+# How to Intergrade with Tower
```
Comment (ihciah) [inline docs/en/how-integrate-with-tower.md]: Integrate?

Comment (pymongo) [reply docs/en/how-integrate-with-tower.md]: Oh my mistake, I fix it

**Review by ihciah — APPROVED** [review]

**Pull request merged**